CYD-Klipper/src/ui/ip_setup.cpp

```diff
@@ -62,6 +62,20 @@ static void keyboard_event_ip_entry(lv_event_t * e) {
     lv_obj_t * ta = lv_event_get_target(e);
     lv_obj_t * kb = (lv_obj_t *)lv_event_get_user_data(e);
 
+    if (code == LV_EVENT_FOCUSED || code == LV_EVENT_DEFOCUSED && ta != NULL)
```

Comment:
Actually, just to make sure, what's the order of operations here? 

Is it supposed to be 
code == LV_EVENT_FOCUSED || (code == LV_EVENT_DEFOCUSED && ta != NULL)

or

(code == LV_EVENT_FOCUSED || code == LV_EVENT_DEFOCUSED) && ta != NULL

because it's currently the former

Reply:
fixed, good catch. I thought `&&` and `||` were equal precedence.